src/renderer/redux/selectors/media.js

```diff
@@ -1,7 +1,7 @@
-import * as settings from 'constants/settings';
 import { createSelector } from 'reselect';
-import { makeSelectClaimForUri } from 'redux/selectors/claims';
+import { makeSelectClaimForUri } from 'lbry-redux';
 
+// eslint-disable-next-line no-underscore-dangle
 const _selectState = state => state.media || {};
```

Comment:
Underscore dangles shouldn't be used for variables or constants. Please, refactor rename the variable as `selectState`.